### ISSUE TITLE
Plasma Heater Deuterium labels don't render properly

### ISSUE THREAD

**Issue by kmecpp** [halvors/Quantum#56]
![image](https://user-images.githubusercontent.com/6520738/45598085-d0e0c780-b9a3-11e8-8371-9a6f8312e812.png)

**Comment by nictrace** [comment]
@halvors , I think we need making a GUI for heater

**Comment by halvors** [comment]
@nictrace Yeah i've considered it, reason it is not made yet is because of the original doesn't have it.
But makes sense to make yes.

**Comment by halvors** [comment]
@kmecpp What mc version?

**Comment by kmecpp** [comment]
1.7.10

**Comment by halvors** [comment]
@nictrace Want to make a GUI for this? Do it in the 1.10 branch and well backport the change.
